README.md

```diff
@@ -7,7 +7,13 @@
 A Rust version of the [libsignal-service-java][lsj] library for communicating
 with Signal servers.
 
-### License
+## Contributing
+
+We're actively trying to make `libsignal-service-rs` fully functional.
+
+If you're looking to contribute, you're more than welcome to join our development channel on Matrix (#whisperfish:rubdos.be) or Freenode (#whisperfish) to get in touch with us!
```

Comment:
```suggestion
If you're looking to contribute or want to ask a question, you're more than welcome to join our development channel on Matrix (#whisperfish:rubdos.be) or Freenode (#whisperfish) to get in touch with us!
```

Reply:
This okay with you, @gferon ?  If so, feel free to squash & merge!

Reply:
Perfect, thanks!